docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-single-proc.adoc

```diff
@@ -0,0 +1,35 @@
+[id='environment-authoring-single-proc']
+= Deploying a single authoring environment
+
+To deploy a single authoring environment, use the `rhpam-70-authoring.yaml` template file. You can download this file from `{URL_BASE_GITHUB_PAM}/templates`.
+
+.Procedure
+
+. Use one of the following methods to deploy the template:
+.. In the OpenShift Web UI, select *Add to Project >  Import YAML / JSON* and then select or paste the `rhpam-70-authoring.yaml` file. In the *Add Template* window, ensure *Process the template* is selected and click *Continue*.
+.. To use the OpenShift command line console, prepare the following command line:
++
+[subs="verbatim,macros"]
+----
+oc process -n <project-name> -f <template-path>/hpam-70-authorimg.yaml -v ATTRIBUTE=value -V ATTRIBUTE=value... | oc create -n <project-name> -f -
+----
++
+In this command line:
++
+* Replace `<project-name>` with the name of your project
+* Replace `<template-path>` with the path to the downloaded template file.
+* Use as many `ATTRIBUTE=value` pairs as needed to set the required attributes. You can view the template file to see descriptions for the attributes.
++
+. Set the following mandatory attributes:
+** *Application Name* (`APPLICATION_NAME`): the name of the OpenShift application. It is used in the default URLs for {CENTRAL} and {KIE_SERVER}.
+** *Business Central Server Certificate Name* (`BUSINESS_CENTRAL_HTTPS_NAME`): the name of the certificate in the keystore that you created in <<secrets-central-create-proc>>.
+** *Business Central Server Keystore Password* (`BUSINESS_CENTRAL_HTTPS_PASSWORD`): the password for the keystore that you created in <<secrets-central-create-proc>>.
+** *KIE Server Certificate Name* (`KIE_SERVER_HTTPS_NAME`): the name of the certificate in the keystore that you created in <<secrets-kie-create-proc>>.
+** *KIE Server Keystore Password* (`KIE_SERVER_HTTPS_PASSWORD`): the password for the keystore that you created in <<secrets-kie-create-proc>>.
+** *KIE Server ID* (`KIE_SERVER_ID`): the name of the server template on the {CENTRAL} that the {KIE_SERVER} is to join. Set this parameter to any value. If you do not set it and then the {KIE_SERVER} pod is restarted or scaled, each new instance of the server might join a new template. 
+** *ImageStream Namespace* (`IMAGE_STREAM_NAMESPACE`): the namespace where the image streams are available. If the image streams were already available in your OpenShift environment (see <<imagestreams-file-install-proc>>), the namespace is `openshift`. If you have installed the image streams file, the namespace is the name of the OpenShift project.
++
+You can also set other parameters as necessary.
```

Comment:
I think that it would be helpful to mention also properties for username and password configuration (KIE_ADMIN_USER and KIE_ADMIN_PASSWORD, KIE_SERVER_USER and KIE_SERVER_PASSWORD, ...). These properties are not mandatory, but if passwords are generated if they are not set.

This should be same for all build steps.

Reply:
Done, except the immutable scenario for now, as I need to ensure I get it right there and it will require a little experimenting. Thanks for pointing this one out @jakubschwan